### PR TITLE
fix/header

### PR DESCRIPTION
**Pull request by temangi**
Issue #:
[Trello](https://trello.com/b/8mLc1U3Y/solid-devs)

## PR type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Other (please describe):

#### before:
Снимок экрана 2023-05-05 в 15 44 41

#### after:
Снимок экрана 2023-05-10 в 20 19 43

## Does this introduce a breaking change?

- [x] Yes (please describe):
- [ ] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [ ] Responsive layout and functionality have been tested
  - [ ] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -7,91 +7,183 @@ import { useRouter } from "next/router";
 import Button from "../Button/Button";
 import i18n from "i18next";
 import { navs } from "@/constants/header";
+import { modal_navs } from "@/constants/header";
+import { cntModal_inputs } from "@/constants/header";
+import krestik from "/public/images/Header/krestic.svg";
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:12]: назови по другому а не крестик

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -7,91 +7,183 @@ import { useRouter } from "next/router";
...
-        >
-          <p onClick={change}>{languageSecondFull}</p>
-          <p>{languageFull}</p>
-        </label>
-      )
-    );
-  }, [isOpen]);
 
-  const headerNavs = useMemo(
+  const cnt_inputs = useMemo(
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:83]: не понятное название и не по BEM

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -7,91 +7,183 @@ import { useRouter } from "next/router";
...
-                  : scss.header__nav_notActive
-              }
-            >
-              <p>{t(text)}</p>
-            </Link>
-          )
-        )
-      ),
-    [route, isOpen]
+      cntModal_inputs.map(({ text, id, name }) => (
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:85]: не по BEM и не понятное

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -7,91 +7,183 @@ import { useRouter } from "next/router";
...
+      <header
+        className={
+          isCntModalOpen
+            ? scss.header__cntModal_isActive
+            : scss.header__cntModal_notActive
+        }
+        onClick={handleButtonClick}
+      >
+        <section
+          className={scss.header__cntModal}
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:108]: не понятное наименование класса

File: src/components/Header/Header.jsx [hunk 5]
```diff
@@ -7,91 +7,183 @@ import { useRouter } from "next/router";
...
       </aside>
       <aside className={scss.header_right}>
-        <div className={scss.header__language} onClick={handleClick}>
-          <p style={{ marginRight: isRussianLanguage ? "40px" : "45px" }}>
-            {language}
+        <div className={scss.header__language}>
+          <p
+            onClick={en}
+            className={
+              !lan ? scss.header__lan_isActive : scss.header__lan_notActive
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:159]: не isActive а просто active

File: src/components/Header/Header.jsx [hunk 6]
```diff
@@ -7,91 +7,183 @@ import { useRouter } from "next/router";
...
+            }
+          >
+            Ру
           </p>
-          <Image src={arrow} alt="arrow" width={16} height={8} />
         </div>
-        <Button title={t("header.button")} withArrow={false} />
-        {languageSelect}
+        <label onClick={handleButtonClick}>
+          {" "}
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:174]: удали

File: src/constants/header.js [hunk 7]
```diff
@@ -6,9 +6,45 @@ export const navs = [
   {
     link: "/carier",
     text: "header.career",
+  }
+];
+
+export const  modal_navs = [
+  {
+    text : "header.mvp",
```
Comment (Improove3567) [inline src/constants/header.js:14]: назови как title

File: src/constants/header.js [hunk 8]
```diff
@@ -6,9 +6,45 @@ export const navs = [
...
+    link : "/mvp"
```
Comment (Improove3567) [inline src/constants/header.js:15]: назови как path

File: src/pages/CarierPage.jsx [hunk 9]
```diff
@@ -5,9 +5,9 @@ import CarierInfoBlock from "@/components/carierInfoBlock/CarierInfoBlock";
 const CarierPage = () => {
   return (
     <>
-      <Header />
-      <CarierInfoBlock variant="team" />
-      <CarierInfoBlock variant="practicum" />
+      <Header/>
+      <CarierInfoBlock variant='team' />
```
Comment (Improove3567) [inline src/pages/CarierPage.jsx:9]: запускай yarn lint, у нас только двойные кавычки

Comment (temangi) [reply src/pages/CarierPage.jsx:9]: это не я делал

File: src/components/Header/Header.jsx [hunk 10]
```diff
@@ -1,97 +1,213 @@
...
-            </Link>
-          )
-        )
-      ),
-    [route, isOpen]
+      inputs.map(({ title, id, name, type }) => (
+        <input
+          placeholder={t(title)}
+          type={type}
+          key={id}
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:102]: key=`${title}_${index}` такой поставь

File: src/components/Header/Header.jsx [hunk 11]
```diff
@@ -1,97 +1,213 @@
...
 import scss from "./Header.module.scss";
 import Link from "next/link";
 import { useRouter } from "next/router";
 import Button from "../Button/Button";
 import i18n from "i18next";
-import { navs } from "@/constants/header";
+import { header__navs } from "@/constants/header";
+import { service__navs } from "@/constants/header";
+import { inputs } from "@/constants/header";
+import close from "/public/images/Header/krestic.svg";
```
Comment (Improove3567) [inline src/components/Header/Header.jsx:12]: переименуй саму картинку тоже как close

**Review by Improove3567 — APPROVED** [review]

**Pull request merged**